doc/en/customize.rst

```diff
@@ -346,3 +346,21 @@ passed multiple times. The expected format is ``name=value``. For example::
         # content of pytest.ini
         [pytest]
         console_output_style = classic
+
+
+.. confval:: empty_parameterset
+
+	.. versionadded:: 3.4
+
+	allows to pick the action for empty parametersets in parameterization
+
+	* ``skip`` skips tests with a empty parameterset
+	* ``xfail`` marks tests with a empty parameterset as xfail(run=False)
+
+	The default is ``skip``, it will be shifted to xfail in future.
```

Comment:
I think mentioning that skip is default in its own description is slightly better:

```rst
* ``skip`` skips tests with a empty parameter set (default)
```

And use this paragraph to mention that we plan to change the default in a future release with a link to #3155